internal/scheduler/scheduler.go

```diff
@@ -195,70 +209,96 @@ func (s *Scheduler) start() {
 
 func (s *Scheduler) runJob(r *job.Run) error {
 	jobId := JobId(r.JobId)
-	j, ok := s.registeredJobs.Load(jobId)
-	job := j.(Job)
-	s.logger.Debug(fmt.Sprintf("starting job run %q for job (%v: %v)", r.PrivateId, jobId, job.Name()))
+	regJob, ok := s.registeredJobs.Load(jobId)
+	j := regJob.(Job)
+	s.logger.Debug("starting job run", "run id", r.PrivateId, "job id", jobId, "name", j.Name())
```

Comment:
Will this panic if `ok` is false?

Reply:
It will, however I was not worried about it because the map is private and controlled by the loop so it will never not be `ok`.  I guess it doesn't hurt to add the check though...

Reply:
The check is already on line 215

Reply:
Oh I see what you are saying, my bad!  I thought you where taking about `j, ok := regJob.(Job)` vs `j := regJob.(Job)`